style.css

```diff
@@ -0,0 +1,733 @@
+    grid-area: 3/1/4/2;
+  }
+
+  .card2 {
+    grid-area: 4/1/5/2;
+  }
+
+  .card2 .card-container,
+  .card4 .card-container {
+    flex-direction: row-reverse;
+  }
+}
+
+/* This is about myself section  */
+
+.about-myself {
+  font-family: 'Poppins', sans-serif;
+  height: auto;
+  background-color: #ffff;
+  border-top-right-radius: 80px;
+}
+
+.about-myself .contact button {
+  font-size: 18px;
+  font-weight: 500;
+  width: 162px;
+  height: 48px;
+  color: #396df2;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  margin: 20px 0;
+  cursor: pointer;
+}
+
+.about-myself .contact button:hover {
+  background-color: #396df2;
+  color: #fff;
+  transition: all 0.5s;
+}
+
+.details {
+  padding-bottom: 30px;
+}
+
+.details .div-container {
+  padding-bottom: 10px;
+  margin: 1em auto;
+}
+
+.frameworks-div .frameworks,
+.skills-div .skills {
+  display: none;
+}
+
+.details .details-header {
+  width: 90%;
+  height: 50px;
+  line-height: 50px;
+  display: flex;
+  justify-content: space-between;
+  padding: 1em auto;
+}
+
+.div-container.hide {
+  border-bottom: 1px solid #999;
+}
+
+.details-header h2,
+.details-header span {
+  cursor: pointer;
+}
+
+.right {
+  transform: rotate(-90deg);
+}
+
+.details ul {
+  list-style-type: none;
+}
+
+.details ul li {
+  font-size: 18px;
+  line-height: 20px;
+  width: 100%;
+  height: 67px;
+  color: #253858;
+  background-color: #f7f7f9;
+  margin: 15px auto;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.details ul li img {
+  width: 50px;
+  height: 50px;
+  background-color: lightgreen;
+  border-radius: 50%;
+  margin: auto  1em;
+}
+
+/* for smaller screen view  */
+@media screen and (max-width: 375px) {
+  .about-myself h1 {
+    font-size: 35px;
+    min-width: 100px;
+  }
+
+  .about-myself p {
+    font-size: 20px;
+    width: 80%;
+    min-width: 270px;
+  }
+
+  .about-myself .connect {
+    font-size: 18px;
+  }
+}
+
+/* For desktop view  */
+@media screen and (min-width: 768px) {
+  .about-myself {
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .about-myself .contact {
+    width: 40%;
+  }
+
+  .about-myself .contact p {
+    color: #344563;
+  }
+
+  .about-myself .contact .connect {
+    font-weight: 500;
+    color: #7f8cff;
+  }
+
+  .about-myself .details {
+    width: 50%;
+    margin-right: 5%;
+  }
+
+  .details ul {
+    display: flex;
+    width: 80%;
+    min-width: 414px;
+    height: auto;
+    text-align: center;
+  }
+
+  .details ul li {
+    width: 122px;
+    height: 120px;
+    border-radius: 10px;
+    display: flex;
+    flex-wrap: wrap;
+    padding: 20px;
+    text-align: center;
+  }
+
+  .details ul li img {
+    margin: auto 0;
+  }
+}
+
+/* This is contact me section  */
+
+.contact-me {
+  clear: both;
+  font-family: 'Poppins', sans-serif;
+  color: #fff;
+  background-color: #6070ff;
+  border-top-left-radius: 100px;
+  text-align: center;
+  padding: 20px 0;
```

Comment:
Kindly consider adding a background image shape to 'contact-me' to make it look like the provided Figma design 😁. Since we are practicing our skills for transforming design into code, following the design as much as possible in this project would be important 💯.

Left: Figma Design; Right: Your Design

![Screenshot 2023-11-25 120150](https://github.com/HsuMyatPhoo/PortfolioFinishMobileVersion/assets/102011195/a26a9ef7-96f3-425d-a65b-851fe4751010)

Reply:
You can update the contact section later since it is okay not to include this section in this week's requirements 😁.